### PR TITLE
Feature/beneficiary applications card

### PR DESCRIPTION
**Pull request by Ajiva-D**

### CODE REVIEW

File: packages/app/helper/modalHelpers.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const toggleModal = (
   dispatch: React.Dispatch<any>,
 ) => {
   if (!localStorage.getItem(key)) {
-    if (modalType === ModalType.SingleAction) {
+    if ((modalType = ModalType.SingleAction)) {
```
Comment (arne-z) [inline packages/app/helper/modalHelpers.ts:19]: This file should not be included in this PR, revert please.

**Review by arne-z — CHANGES_REQUESTED** [review]

File: packages/app/helper/modalHelpers.ts (hunk 1 above)
Comment (Ajiva-D) [reply packages/app/helper/modalHelpers.ts]: Done

**Review by arne-z — APPROVED** [review]
